Posh-ACME/Plugins/Infomaniak.ps1

```diff
@@ -255,8 +256,6 @@ Function Find-InfomaniakZone {
         $zoneTest = $zoneTest.Split('.',2)[1]
     }
 
-    # remove _acme-challenge. from requested zone
-    $zone = $RecordName.Split('.',2)[1]
     Write-Debug "Zone $zone does not exist ..."
```

Comment:
Whoops. Now the debug message references a `$zone` variable that doesn't exist.

Reply:
Right. I'll stop programming today. Already too late. I will review the whole thing in a few hours.

Reply:
No worries. Take your time.